pydata/utils.py

```diff
@@ -0,0 +1,30 @@
+from workshops.models import Tag, Event, Role
+
+
+def parse_event(conf):
+    return {
+        'slug': conf['title'],
+        'start': conf['start_date'],
+        'end': conf['end_date'],
+        'tags': Tag.objects.get(name='PyData').pk,
+    }
+
+
+def parse_person(speaker):
+    personal = speaker['name'].rsplit(' ', 1)[0]
+    family = speaker['name'].rsplit(' ', 1)[-1]
+    return {
+        'personal': personal,
+        'family': family,
+        'email': speaker['email'],
+    }
+
+
+def parse_task(presentation, conf_url):
+    event = Event.objects.get(url__contains=conf_url)
```

Comment:
If we allow `EventLookup` to search against the `url` field, we wouldn't have to fetch the Event from database and pass it's `pk` within the response.